### ISSUE TITLE
does the audio have a transcript?

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#3935]
just as images need alt-text, audio does too.  

Janeway needs new functionality to allow for text to be attached to all non-text media in accordance with WCAG.

**Comment by StephDriver** [comment]
from discussion with @ajrbyers 

@StephDriver to investigate the requirements and how they relate to what we need to do.  Be specific about what we need to do in order to be compliant.

**Comment by StephDriver** [comment]
# A11y Requirements

## Summary

Standard: WCAG 2.2 AA.
- [Section 1.1 Text Alternatives](https://www.w3.org/TR/WCAG22/#text-alternatives)
- [Section 1.2 Time-based Media](https://www.w3.org/TR/WCAG22/#time-based-media)

Media: 
- Pre-recorded audio-only
- live audio-only
- mixed video and audio media

Sound effects, clicks, bleeps etc that are of short duration and decorative rather than either conveying information or intending to create a specific sensory experience are outside the scope of this issue. The audio under discussion here is more likely to be a podcast, the sound-track to a video presentation or something else which makes a substantial contribution to what is on that page.

## Details

### Types of Audio
- non-verbal audio (e.g. music, sound effects)
   - used to set a mood or `create a specific sensory experience`. These are not purely decorative, but also cannot be transcribed directly into words.  
- verbal

**Comment by StephDriver** [comment]
## Questions arising
- how will adding a text transcript to audio look in the back office?
    -  Where does it fit into the workflow?
    - will it be a file upload, or edited within Janeway?
    - does a transcript need to go through the standard workflow - copyediting, typesetting etc?
        - should this be optional? consider the 30 min podcast news-item versus a 2 min illustration within a long text article. 
        - do we anticipate multimedia articles in the future?

**Comment by ajrbyers** [comment]
We held off discussing this issue at the 18/04 refinement meeting so that we can discuss it with @StephDriver.

**Comment by joemull** [comment]
Backlog refinement:

@mauromsl to break down some of this into dev issues and then assign to @StephDriver to look at the accessibility affordances.
